README.md

```diff
@@ -356,6 +356,8 @@ Beyond implementing various generative modeling inference and training pipelines
 - [diffusers](https://github.com/huggingface/diffusers): training and inference pipelines for generative models from [Diffusers Models](https://github.com/huggingface/diffusers)
 - [GFlowNets](https://github.com/recursionpharma/gflownet): training and inference pipeline for [Generative Flow Networks](https://yoshuabengio.org/2022/03/05/generative-flow-networks/)
 
+- [MolGX](https://github.com/GT4SD//molgx/): training and inference pipelines to generate small molecules satisfying target properties. The full functions of MolGX are available [here](https://github.com/GT4SD/molgx/). 
```

Comment:
```suggestion
- [MolGX](https://github.com/GT4SD//molgx/): training and inference pipelines to generate small molecules satisfying target properties. The full implementation of MolGX, including additional functionalities, is available [here](https://github.com/GT4SD/molgx/). 
```

Reply:
I have also noticed an incorrect link to MolGX so have fixed that issue at the same time, while reflecting the above change.